barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.test.cpp

```diff
@@ -44,6 +48,142 @@ TEST_F(IPATest, CommitOnManyZeroCoeffPolyWorks)
     EXPECT_EQ(expected.normalize(), commitment.normalize());
 }
 
+TEST_F(IPATest, OpenZeroPolynomial)
```

Comment:
can you add brief comments explaining the why behind each test

Reply:
done